app/src/main.ts

```diff
@@ -35,6 +36,13 @@ async function init() {
 	app.use(i18n);
 	app.use(createPinia());
 
+	app.config.errorHandler = (err, vm, info) => {
+		const source = getVueComponentName(vm);
+		console.warn(`[app-${source}-error] ${info}`);
+		console.warn(err);
+		return false;
+	};
```

Comment:
Lets use the `unexpectedError` dialog here as well. We might even want to add a `fatal: true` flag to `unexpectedError` that forces the user to refresh the browser just in case.

Reply:
I stand corrected 🫡